### PR TITLE
CONTRIBUTING: add testing viewer build instructions

### PR DESCRIPTION
**Pull request by ebidel**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,7 +38,14 @@ echo "Test the CLI."
 yarn run smoke
 echo "Test a fresh local install (see below)"
 echo "Test the extension"
+
 echo "Test the lighthouse-viewer build"
+# Manual test for now:
```
Comment (brendankenny) [inline CONTRIBUTING.md]: move this down with `# * Testing a fresh local install *`

Comment (ebidel) [inline CONTRIBUTING.md]: I had it there but I think it makes more sense to keep next to the reminder echo.

Comment (brendankenny) [inline CONTRIBUTING.md]: do you want to move up the local install thing then?

Comment (paulirish) [inline CONTRIBUTING.md]: done

**Pull request merged**